gen3-client/g3cmd/utils.go

```diff
@@ -290,33 +290,36 @@ func validateFilePath(filePaths []string, forceMultipart bool) ([]string, []stri
 		}
 
 		file, err := os.Open(filePath)
-		defer file.Close()
```

Comment:
would this work?
```
file, err := os.Open(filePath)
if err != nil {
  log.Println("File open error occurred when validating file path: " + err.Error())
  continue
}
defer file.Close()
```

Reply:
no because `defer` will be executed when the function returns...

probably a better way to write is to put whatever inside that for loop into an anonymous function? I can try that approach

Reply:
fixed in https://github.com/uc-cdis/cdis-data-client/pull/57/commits/673dcd26f0938a66b4edf357978d698f3c27cff7